src/desktop/apps/personalize/routes.js

```diff
@@ -2,37 +2,59 @@ import * as _ from "underscore"
 import { stitch } from "@artsy/stitch"
 import { App } from "desktop/apps/personalize/components/App"
 
+const computeStitchOptions = (request, response) => {
+  const basePath = request.app.get("views")
+  const currentUser = response.locals.sd.CURRENT_USER
+  const redirectTo = request.query.redirectTo
+  const forceStep = request.params.slug
+
+  const options = {
+    basePath,
+    config: {
+      styledComponents: true,
+    },
+    layout: "../../components/main_layout/templates/react_blank_index.jade",
+    blocks: {
+      head: "./meta.jade",
+      body: App,
+    },
+    locals: {
+      ...response.locals,
+      assetPackage: "onboarding",
+    },
+    data: {
+      currentUser,
+      forceStep,
+      redirectTo,
+      title: "Personalize | Artsy",
+    },
+  }
+
+  return options
```

Comment:
As yr one of the few people I notice who do this, want to call out some props: its always nice to see folks `return` self-describing variables, vs logical operations, etc 👍 Makes code much easier to read and debug in the future.